roles/network/tasks/hostapd.yml

```diff
@@ -144,3 +144,8 @@
     regexp: '^HOSTAPD_ENABLED=*'
     line: 'HOSTAPD_ENABLED={{ hostapd_enabled }}'
     state: present
+
+- name: Object to local_vars where wifi_up_down is true and hostapd_enabled is false
+  pause:
+    prompt: "Please enable hostapd_enabled if you want wifi_up_down: True"
+  when: not hostapd_enabled and wifi_up_down
```

Comment:
Why? What if the user wants upstream wifi but no AP, this should be a valid use case.